tsc/process/release.md

```diff
@@ -0,0 +1,20 @@
+**Release Process for OpenVDB**
+
+The following assumes that the current OpenVDB library version number is 6.0.0 and the new version number is 6.1.0. Adjust for the actual version numbers as appropriate.
+
+- [ ] Open a Jira "Release OpenVDB 6.1.0" ticket with "OpenVDB_6.1.0" as the Fix Version.
+- [ ] Update `openvdb/CHANGES` and `openvdb/doc/changes.txt` with release notes.
+- [ ] Update `openvdb/INSTALL` with the library version number and any directory hierarchy changes.
+- [ ] Push the above changes to `openvdb/master`, associating them with the Jira ticket created earlier, and verify that the Travis CI build runs successfully.
```

Comment:
Our policy is no direct pushes to master, so I think this should still be a PR which is then approved before being merged in.

Reply:
Agreed—I've added that clarification.